docs/stacks-guide-aws.md

```diff
@@ -61,49 +63,49 @@ INFRA_NAMESPACE=infra-aws
 kubectl create namespace ${INFRA_NAMESPACE}
 ```
 
-The output will look like the following:
-
-```bash
-namespace/infra-aws created
-```
-
-Now we can create the AWS stack by using Crossplane CLI.  Since this is an
-infrastructure stack, we need to specify that it's cluster-scoped via
-`--cluster` flag.
+Now we can install the AWS stack using Crossplane CLI.  Since this is an
+infrastructure stack, we need to specify that it's cluster-scoped by
+passing the `--cluster` flag.
 
 ```bash
 kubectl crossplane stack generate-install --cluster 'crossplane/stack-aws:master' stack-aws | kubectl apply --namespace ${INFRA_NAMESPACE} -f -
 ```
 
-The next steps assume that you installed the AWS stack into the `infra-aws`
-namespace.
+The rest of the steps assume that you installed the AWS stack into the
+`infra-aws` namespace.
```

Comment:
i'm not sure we're consistent across all the guides for the naming of namespaces, but i wouldn't change that at this point

Reply:
Yeah. I noticed the same thing and made the same call